packages/twenty-server/src/engine/api/graphql/workspace-schema-builder/services/type-mapper.service.ts

```diff
@@ -146,7 +146,7 @@ export class TypeMapperService {
       );
     }
 
-    if (!options.nullable && !options.defaultValue) {
+    if (options.nullable === false && options.defaultValue === null) {
```

Comment:
@magrinj FYI! the isRequired computation for boolean having defaultValue false was leading to errors